.prettierrc.json

```diff
@@ -1 +1,7 @@
-{}
+{
+  "printWidth": 120,
```

Comment:
🔶 Why are we changing the default print width?

Reply:
To be honest, I think 80 is too little. It's also what we use on Louis.

Reply:
80 is too little, it's a limit based on outdated display constraints

Reply:
This one will have to be on multiple new lines despite in reality the checking is simple.

```
const SomeComponent = () => {
    // New lines on default 80 chars width, looks ugly and hard to read:
    if (
        firstLongNameHelper(longvalue1, longvalue2)
        && secondLongNameHelper(longvalue3, longvalue4)
    ) {
        ...
        // do something
    }

    // Same line, easier to skim and check:
    if (firstLongNameHelper(longvalue1, longvalue2) && secondLongNameHelper(longvalue3, longvalue4)) {
        // do something
    }
...
};
```

Reply:
I learned on 60 😆

But seriously, I think we need a good case to diverge from the default.

Reply:
In Sang's example, I prefer the multiline version

Reply:
But we don't code on Github comment. We have wider screens. :(

Reply:
My argument is that I value vertical space almost more than horizontal and I rarely have ran out of horizontal space. Shorter lines means more vertical space used.

Reply:
But this is subjective, and I'd rather stick to the default, unless a clear objective argument can be made

Reply:
And it's weird to talk about assuming everyone has wide screens, when earlier today we were talking about developers who don't use a screen at all

Reply:
I don't think we should. What's to prevent someone asking for 1000 screenwidth? Should we compromise at 500?